### ISSUE TITLE
[FEATURE_REQUEST] Docker should allow command arguments passthrough

### ISSUE THREAD

**Issue by FallingSnow** [SillyTavern/SillyTavern#1672]
### Have you searched for similar requests?

No

### Is your feature request related to a problem? If so, please describe.

Cannot pass through command line flags to docker instance.

### Describe the solution you'd like

Change `docker-compose.yaml` to define `init: true` instead of using tini inside of the docker container. [Remove tini entrypoint](https://github.com/SillyTavern/SillyTavern/blob/7104daf7f092735331d17f5d2e62e211c110c0c0/Dockerfile#L10) and tini install. Change `Dockerfile` to use `ENTRYPOINT` instead of [`CMD`](https://github.com/SillyTavern/SillyTavern/blob/7104daf7f092735331d17f5d2e62e211c110c0c0/Dockerfile#L51). Add argument passthrough by [changing the entrypoint script to accept additional arguments](https://github.com/SillyTavern/SillyTavern/blob/7104daf7f092735331d17f5d2e62e211c110c0c0/docker/docker-entrypoint.sh#L38) (Change to `exec node server.js $@`).

Once this is done you should be able to start ST with arguments like `--ssl` by adding `command: --ssl` to your `docker-compose.yaml` file.

### Priority

Low (Nice-to-have)

### Is this something you would be keen to implement?

Maybe

**Comment by FallingSnow** [comment]
I've tested the changes I've suggested and they work.

To anyone who wants this in the future, you'll also need to make a certs directory and bind it to your container. Then create a cert and key in that directory using `openssl req -nodes -x509 -newkey rsa:4096 -days 3650 -keyout privkey.pem -out cert.pem`.
